src/main.js

```diff
@@ -6,6 +6,18 @@ var fs = require('fs');
 
 require('crash-reporter').start();
 
+// Load persisted state.
+var state = {};
+var statePath = path.join(app.getDataPath(), "state.json");
+try {
+  state = JSON.parse(fs.readFileSync(statePath, 'utf8'));
+} catch(e) {
+}
```

Comment:
Add simple comment explaining why this block is intentionally left blank and why Pokémon exception handling works.